pkg/controllers/node_controller.go

```diff
@@ -25,7 +25,7 @@ const (
 	RedisIDsEnv             = "REDIS_IDS"
 	ReservedIPsPoolEnv      = "RESERVED_IPS_POOL"
 	SecurityGroupIDs        = "SECURITY_GROUP_IDS"
-	NumberRetries           = "NUMBER_RETRIES"
+	NumberRetries           = "RETRIES_NUMBER"
```

Comment:
Could you modify it to still support `NUMBER_RETRIES` ? In order to not do any breaking changes

Reply:
Done! I've edited the configmap instead, since the `RETRIES_NUMBER` wasn't used by the code anyway AFAIK :thinking: